### PR TITLE
[OneExplorer] Show profile result from profileModel command

### PR DESCRIPTION
**Pull request by dayo09**
This commit shows profile result file created by profileModel command.

- ONE Explorer basically shows a file written in config.json. If it's not written in config.json, it won't be shown.
- one.toolchain.profileModel command generates <model>.trace.json in the same directory. Let's show it exceptionally.

ONE-vscode-DCO-1.0-Signed-off-by: Dayoung Lee <dayoung.lee@samsung.com>

### CODE REVIEW

File: src/OneExplorer/ConfigObject.ts [hunk 1]
```diff
@@ -325,6 +325,24 @@ export class ConfigObj {
       }),
     });
 
+    // NOTE
+    // Shows <model>.trace.json
+    // REQUIRES: <model>.tvn be written in the config file.
```
Comment (dayo09) [inline src/OneExplorer/ConfigObject.ts:330]: @jyoungyun The tvn file should be written in the config file to show it. Will it be enough?

Comment (jyoungyun) [reply src/OneExplorer/ConfigObject.ts:330]: Sure. I'm considering profiling with the tvn file created in ONE-VSCode, so it is satisfied with your constraints.

**Review by jyoungyun — APPROVED** [review]
LGTM

**Review by seanshpark — APPROVED** [review]
LGTM

**Pull request merged**